mshadow

```diff
@@ -1 +1 @@
-Subproject commit bc5cb993627f7a6633251f752ac93b0157e9f085
+Subproject commit 74be312ab6f20178766901a7caf021d4829e9110
```

Comment:
this is reason why it is failed. to fix it, you changed mshadow to an older version. to fix it

```
cd mshadow
git pull
```

and then merge the changes of mshadow (which is a pointer to a commit)